plv8_func.cc

```diff
@@ -102,7 +102,7 @@ SetCallback(Handle<ObjectTemplate> obj, const char *name,
 			FunctionCallback func, PropertyAttribute attr = None)
 {
 	Isolate* isolate = Isolate::GetCurrent();
-	obj->Set(String::NewFromUtf8(isolate, name, String::kInternalizedString),
+	obj->Set(String::NewFromUtf8(isolate, name, NewStringType::kInternalized).ToLocalChecked(),
```

Comment:
I don't think we need to actually use `NewStringType` in these places, since it gracefully degrades.

Reply:
Ah, yeah, it's left from earlier versions. In newer ones they even have convenience functions for that.  Need to update...

Reply:
it's not just a need to update, but in some versions of v8 it actually produces bad strings.  very frustrating.  removing it fixes the bad string creation.  typically, it's something like `data\u37`.  so it appears to be bad unicode conversion in v8 itself.   I believe they've fixed it in more recent versions, but I'd rather not be on the bad side of that when they screw up and revert a patch.

Reply:
Do you want to use Normal string types everywhere? I have changed a lot of places to NewFromUtf8Literal which saves some typing but Internalized strings should behave differently, I think.